mkdocs/docs/HPC/connecting.md

```diff
@@ -759,3 +759,17 @@ created.
 ### Fast file transfer for large datasets
 
 See [the section on `rsync` in chapter 5 of the Linux intro manual](../linux-tutorial/uploading_files/#copying-faster-with-rsync).
+
+## Changing login nodes
+It can be useful to work on the same login node using multiple terminal sessions. However, when you connect to the HPC (High-Performance Computing) system through SSH, you're directed to a random login node, which might not be the one where you already have an active session. To address this, there is a way to manually switch your active login node.
+
+For instance, if you want to switch to the login node named `gligar07`, you can use the following command while you are connected to the HPC:
+```shell
+ssh gligar07
+```
+
+Rather than launching multiple terminals, you can also use a terminal multiplexer like `screen` or `tmux`.
```

Comment:
`screen` and `tmux` are mostly useful when you want to set up a shell session that "survives" across disconnects, not so much to launch multiple sessions

Reply:
I changed this